samples/net/openthread/ncp/overlay-tri-n4m-br.conf

```diff
@@ -0,0 +1,35 @@
+# TRI n4m BR
+# Use UART_0 for Shell/Console
+CONFIG_UART_SHELL_ON_DEV_NAME="UART_0"
+CONFIG_UART_CONSOLE_ON_DEV_NAME="UART_0"
+CONFIG_SHELL_PROMPT_UART="ncp-uart:~$ "
+
+# Use UART_1 for NCP
+CONFIG_UART_MCUX_1=y
+CONFIG_OPENTHREAD_NCP_SPINEL_ON_UART_DEV_NAME="UART_1"
+
+# Use USB-CDC-ACM for NCP
+CONFIG_OPENTHREAD_NCP_SPINEL_ON_UART_ACM=y
+CONFIG_OPENTHREAD_NCP_SPINEL_ON_UART_DEV_NAME="CDC_ACM_0"
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_GPIO=y
+CONFIG_USB=y
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_PRODUCT="OT BorderRouter K64"
+CONFIG_LOG=y
+CONFIG_USB_DRIVER_LOG_LEVEL_ERR=y
+CONFIG_USB_CDC_ACM=y
+CONFIG_USB_DEVICE_LOG_LEVEL_ERR=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_UART_LINE_CTRL=y
+CONFIG_CDC_ACM_IAD=y
```

Comment:
CDC_ACM_IAD is enabled per default and IMO not necessary on linux host

Reply:
removed. this is not a linux host though. it is for an NXP K64 MCU.